extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.ts

```diff
@@ -176,7 +176,7 @@ export class DragAndDropSortInputValidationService {
               }
             }
 
-            if (areAnyItemsEmpty) {
+            if (areAnyItemsEmpty || xInputs.length === 0) {
```

Comment:
Isn't the message kinda confusing for the second case?

Reply:
Done.